buzzmobile/tests/test_utils/launch_tools.py

```diff
@@ -70,22 +77,23 @@ def launcher(func):
         def new_test(self):
             """Wrapper around the user-provided test that runs a ros node.
             """
-            node = roslaunch.core.Node(package, name, namespace=namespace)
+            env = {'ROS_MASTER_URI': self.rosmaster_uri}
+            node = roslaunch.core.Node(package, name, namespace=namespace,
+                    env_args=env.iteritems())
```

Comment:
Unfortunately, I don't *think* so. env args sets environmental variables (equivalent to `os.environ[x]= y` in python, or `export VAR=VALUE` in bash). I'll expand on this in a sec, but it doesn't do anything with command line args.